From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
network watcher: conncetivity

Commit:
updates

Commit:
updates to docs, added to toc:

Commit:
added rest

Commit:
preview notice

Commit:
correcting link

Commit:
corrected vm example

Commit:
corrections to intro

Commit:
correcting table with line breaks

Commit:
corrected various grammar and punctuation

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into connectivity

Commit:
updates

Commit:
updates to connectivity docs

Commit:
additions to connectivity

Commit:
added additional commit

jpallavi committed:
Update backup-azure-restore-key-secret.md

New cmdlet to restore key and secret using JSON.

jpallavi committed:
Update backup-azure-restore-key-secret.md

jpallavi committed:
Update backup-azure-restore-key-secret.md

jpallavi committed:
Update backup-azure-restore-key-secret.md

Commit:
updates to tables

Commit:
custom operation tracking

Commit:
fix broken link

v-awku committed:
USER STORY 1025740 SaaS App Tutorial: Jobscience Tutorial

v-awku committed:
USER STORY 1025740 SaaS App Tutorial: Jobscience Tutorial

v-awku committed:
USER STORY 1025740 SaaS App Tutorial: Jobscience Tutorial

venkatasiriki committed:
USER STORY 1024869 SaaS App Tutorial: Mimecast Personal Portal Update

Commit:
ip02

Commit:
ip02

v-viinde committed:
USER STORY 1030953 SaaS App Tutorial: SECURE DELIVER Tutorial Update

v-viinde committed:
USER STORY 1031103 SaaS App Tutorial: SimpleNexus Tutorial Update

v-nagta committed:
USER STORY 1032701 SaaS App Tutorial: Zscaler One Update

v-nagta committed:
USER STORY 1032702 SaaS App Tutorial: Zscaler Two Update

v-nagta committed:
USER STORY 1032703 SaaS App Tutorial: Zscaler Beta Update

v-viinde committed:
USER STORY 1031105 SaaS App Tutorial: SkyDesk Email Tutorial Update

v-nagta committed:
USER STORY 1032705 SaaS App Tutorial: xMatters OnDemand Update

v-nagta committed:
Updated

v-nagta committed:
Updated

v-nagta committed:
Updated

venkatasiriki committed:
USER STORY 1024869 SaaS App Tutorial: Mimecast Personal Portal Update

v-nagta committed:
Updated

v-nagta committed:
Updated

v-royhar committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

Commit:
updates to preview

v-royhar committed:
Updated and new images

v-royhar committed:
Updated tutorial to show face features including emotion.

v-royhar committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into v-royhar-face-tutorial-cs

clarkio committed:
update step 2 text in "test the function" section

Commit:
update

v-viinde committed:
Updated

Commit:
added portal article

Commit:
update portal doc

clarkio committed:
remove link to edge browser

Commit:
Review comments

Commit:
style fixes

v-nagta committed:
Updated

v-nagta committed:
Updated

v-nagta committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

jpallavi committed:
Update backup-azure-restore-key-secret.md

venkatasiriki committed:
USER STORY 1024869 SaaS App Tutorial: Mimecast Personal Portal Update

linda33wj committed:
Refresh connector content

v-viinde committed:
Updated

v-awku committed:
USER STORY 1025740 SaaS App Tutorial: Jobscience Tutorial

jpallavi committed:
Update backup-azure-restore-key-secret.md

v-nagta committed:
Updated

v-royhar committed:
Minor edits.

v-royhar committed:
More minor edits.

v-royhar committed:
Minor edit.

v-royhar committed:
Minor edit.

CarlRabeler committed:
python

v-royhar committed:
Minor edit.

CarlRabeler committed:
java

Commit:
few minor updates

v-royhar committed:
Minor edit.

Commit:
fix links

v-royhar committed:
Minor edit.

mumian committed:
include excel 2016 info

Commit:
ip02

v-royhar committed:
Minor edit.

v-royhar committed:
Minor edit.

Doing these minor edits to fix a code display issue.

RenaShahMSFT committed:
Updated the documentation  for Encryption support

Mount error 11 was fixed in the encryption support for Ubuntu 16.10 and 17.04. Mount error 115 was partially fixed

v-royhar committed:
Minor edit.

If this one works, it's ready to go.

mumian committed:
format

CarlRabeler committed:
wording

v-royhar committed:
Minor edit.

Commit:
updates to articles

robinsh committed:
Change service from 'managed-disks-windows' to 'managed-disks' to match Compute's articles.

CarlRabeler committed:
redirect

v-royhar committed:
Minor edit.

GitHubber17 committed:
Merge pull request #16801 from v-nagta/zscalerone

USER STORY 1032701 SaaS App Tutorial: Zscaler One Update

GitHubber17 committed:
Merge pull request #16802 from v-nagta/zscalertwo

USER STORY 1032702 SaaS App Tutorial: Zscaler Two Update

GitHubber17 committed:
Merge pull request #16803 from v-nagta/zscalerbeta

USER STORY 1032703 SaaS App Tutorial: Zscaler Beta Update

v-royhar committed:
Minor edit.

Hoping this is the final minor edit.

GitHubber17 committed:
Merge pull request #16162 from v-awku/jobscience

USER STORY 1025740 SaaS App Tutorial: Jobscience Tutorial

GitHubber17 committed:
Merge pull request #16262 from venkatasiriki/mimecastpersonalportal

 USER STORY 1024869 SaaS App Tutorial: Mimecast Personal Portal Update

Commit:
updates

GitHubber17 committed:
Merge pull request #16794 from v-viinde/securedeliver

USER STORY 1030953 SaaS App Tutorial: SECURE DELIVER Tutorial Update

GitHubber17 committed:
Merge pull request #16795 from v-viinde/simplenexus

USER STORY 1031103 SaaS App Tutorial: SimpleNexus Tutorial Update

tynevi committed:
redirecting contributor guide articles

PRMerger-2 committed:
Merge pull request #17096 from robinsh/robinsh-0711-kris

GitHubber17 committed:
Merge pull request #16804 from v-viinde/skydeskemail

USER STORY 1031105 SaaS App Tutorial: SkyDesk Email Tutorial Update

GitHubber17 committed:
Merge pull request #16818 from v-nagta/xmattersondemand

USER STORY 1032705 SaaS App Tutorial: xMatters OnDemand Update

tynevi committed:
Merge pull request #17100 from tysonn/master

redirecting contributor guide articles

GitHubber17 committed:
Merge pull request #17088 from RenaShahMSFT/patch-29

Updated the documentation  for Encryption support

v-stgreg committed:
Removed 2013 version number for Visual Studio Express

genlin committed:
rewording

PRMerger-2 committed:
Merge pull request #17104 from v-stgreg/hdinsight

PRMerger-2 committed:
Merge pull request #16868 from clarkio/patch-3

PRMerger-2 committed:
Merge pull request #17106 from genlin/master013

yushwang committed:
Bugfixes - clarify DH groups, GCMAES constraint, default SA lifetime values, and gateway SKUs

yushwang committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into policyfix

lindydonna committed:
Update functions-run-local, fix date

JasonWHowell committed:
Update quickstart-create-server-database-portal.md

PRMerger-2 committed:
Merge pull request #17112 from lindydonna/patch-1

PRMerger-2 committed:
Merge pull request #17115 from JasonWHowell/patch-6

Commit:
addressed review suggestions

GitHubber17 committed:
Merge pull request #16659 from MarkusVi/ip02

ip02

GitHubber17 committed:
Merge pull request #17110 from yushwang/policyfix

Policyfix

PRMerger-2 committed:
Merge pull request #17027 from linda33wj/master

Commit:
corrected issues

genlin committed:
add the note

genlin committed:
wording

PRMerger-2 committed:
Merge pull request #17117 from genlin/master013

jpallavi committed:
Update backup-azure-restore-key-secret.md

jpallavi committed:
Update backup-azure-restore-key-secret.md

JasonWHowell committed:
Update connect-python.md

JasonWHowell committed:
Update connect-python.md

Commit:
freshness200

Commit:
freshness201

PRMerger-2 committed:
Merge pull request #17118 from JasonWHowell/patch-6

PRMerger-2 committed:
Merge pull request #17122 from MarkusVi/freshness201

PRMerger-2 committed:
Merge pull request #17121 from MarkusVi/freshness200

dominicbetts committed:
Fix metadata issues

dominicbetts committed:
Fix metadata issues

mumian committed:
rename image files

Commit:
Update backup-azure-arm-vms-prepare.md

PRMerger-2 committed:
Merge pull request #17143 from trinadhk/patch-24

Commit:
Update data-lake-analytics-manage-use-portal.md

Commit:
Update data-lake-analytics-manage-use-powershell.md

Commit:
Update data-lake-analytics-manage-use-python-sdk.md

Commit:
Update data-lake-analytics-manage-use-portal.md

MGoedtel committed:
Updated Get Started Article

PRMerger-2 committed:
Merge pull request #17147 from saveenr/patch-4

PRMerger-2 committed:
Merge pull request #17146 from saveenr/patch-3

PRMerger-2 committed:
Merge pull request #17148 from MGoedtel/UpdateAAGetStart7122017

PRMerger-2 committed:
Merge pull request #17145 from saveenr/patch-2

itechedit committed:
Merge pull request #14542 from georgewallace/connectivity

network watcher : Connectivity

billmath committed:
updating freshness

itechedit committed:
Merge pull request #15446 from JPallavi/patch-19

Update backup-azure-restore-key-secret.md

Commit:
removed promise to deliver features

daveirwin1 committed:
Updated DSC example.

PRMerger-2 committed:
Merge pull request #17150 from daveirwin1/master

PRMerger-2 committed:
Merge pull request #17149 from billmath/aadconnect

CarlRabeler committed:
Update sql-database-connect-query-java.md

CarlRabeler committed:
Update sql-database-connect-query-python.md

CarlRabeler committed:
Update sql-database-connect-query-java.md

CarlRabeler committed:
Update sql-database-connect-query-python.md

v-shils committed:
Merge pull request #16866 from v-royhar/v-royhar-face-tutorial-cs

Updated Face API C# tutorial to show face features including emotion.

v-shils committed:
Merge pull request #17040 from dominicbetts/fixmetadata

Fix metadata

v-shils committed:
Merge pull request #17131 from dominicbetts/fixmetadata2

Fix metadata issues (part 2)

v-shils committed:
Merge pull request #15855 from SergeyKanzhelev/customOperationsTracking

custom operation tracking

v-shils committed:
Merge pull request #17060 from CarlRabeler/20170711-python

python

v-shils committed:
Merge pull request #17095 from CarlRabeler/20170711-redirect

redirect

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-shils committed:
Merge pull request #17086 from mumian/excel0711

include excel 2016 info

v-alje committed:
Merge pull request #17154 from MicrosoftDocs/FromPublicRepo

From public repo
